notebooks/postgresql2bq/postgresql-to-bigquery-notebook.ipynb

```diff
@@ -0,0 +1,2413 @@
+    "\n",
+    "POSTGRESQL_TO_BIGQUERY_JOBS = []\n",
+    "# If SERVICE_ACCOUNT is not specified it will take the one attached to Notebook\n",
+    "if SERVICE_ACCOUNT == '':\n",
+    "    shell_output = !gcloud auth list 2>/dev/null\n",
+    "    SERVICE_ACCOUNT = shell_output[2].replace(\"*\", \"\").strip()\n",
+    "    print(\"Service Account: \",SERVICE_ACCOUNT)\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "b25c6c38-df60-4ef6-a893-7bba3a6060fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not IS_PARAMETERIZED:\n",
+    "    POSTGRESQL_HOST = \"\"\n",
+    "    POSTGRESQL_PORT = \"3306\"\n",
+    "    POSTGRESQL_USERNAME = \"\"\n",
+    "    POSTGRESQL_PASSWORD = \"\"\n",
+    "    POSTGRESQL_DATABASE = \"\"\n",
+    "    POSTGRESQL_TABLE_LIST = [] # leave list empty for migrating complete database else provide tables as ['table1','table2']\n",
+    "    POSTGRESQL_SCHEMA_LIST = [] # schema list to migrate. Only Migrate tables associated with the provided schema list\n",
+    "\n",
+    "    BIGQUERY_DATASET = \"\"\n",
+    "    BIGQUERY_MODE = \"\" # output write mode (Default: overwrite)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f129a976-154d-4627-aa08-6430eac9b3fc",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Step 3.2 PostgreSQL Parameters\n",
+    "\n",
+    "POSTGRES_HOST : PostgreSQL instance ip address\n",
+    "\n",
+    "POSTGRES_PORT : PostgreSQL instance port\n",
+    "\n",
+    "POSTGRES_USERNAME : PostgreSQL username\n",
+    "\n",
+    "POSTGRES_PASSWORD : PostgreSQL password\n",
+    "\n",
+    "POSTGRES_DATABASE : name of database that you want to migrate\n",
+    "\n",
+    "OUTPUT_MODE : Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to overwrite)\n",
+    "\n",
+    "BATCH_SIZE : JDBC output batch size. Default set to 1000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "e1aa479e-8318-4c5d-a36b-fdb497a6053a",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "POSTGRESQL_HOST=\"10.0.0.11\"\n",
+    "POSTGRESQL_PORT=\"5432\"\n",
+    "POSTGRESQL_USERNAME=\"my_username\"\n",
+    "POSTGRESQL_PASSWORD=\"my_password\"\n",
+    "POSTGRESQL_DATABASE=\"my_database\"\n",
+    "JDBCTOJDBC_OUTPUT_MODE=\"overwrite\" # one of append/overwrite/ignore/errorifexists\n",
+    "JDBCTOJDBC_OUTPUT_BATCH_SIZE=\"1000\"\n",
+    "POSTGRESQL_TABLE_LIST=[] # leave list empty for migrating complete database else provide tables as ['dbo.table1','sys.table2']. If this parameter is not empty, leave POSTGRESQL_SCHEMA_LIST empty.\n",
+    "POSTGRESQL_SCHEMA_LIST=['public'] # leave list empty for migrating complete database else provide schema as ['schema1','schema2'] for migrating all tables in specific schemas. If this parameter is not empty, leave POSTGRESQL_TABLE_LIST empty."
```

Comment:
This looks like a repeat of variables declared in the previous code cell, right?